e2e/tests/examples/data-driven.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { config } from 'resources/workbench-config';
+import GoogleLoginPage from 'app/page/google-login';
+import { withPage } from 'libs/page-manager';
+
+describe('Login tests:', () => {
+  const urls = [
+    { text: `${config.uiBaseUrl}${config.workspacesUrlPath}` },
```

Comment:
this keyname does not seem accurate. In this case I would probably just provide an array of strings rather than objects, but if you're establishing a pattern, a key of `url` or `value` is probably more appropriate. Desctructuring into the below callback would also be good.

Reply:
Good points. I changed `text` string to `value`.